containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzServer.java

```diff
@@ -0,0 +1,107 @@
+/*
+ *
+ *  * Copyright The Athenz Authors
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package com.yahoo.athenz.container;
+
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import java.security.KeyStore;
+import java.security.cert.*;
+import java.util.Collection;
+
+public class AthenzServer extends SslContextFactory.Server {
+
+    private static final Logger LOG = LoggerFactory.getLogger("SSLConnectionLog");
+
+    @Override
+    protected TrustManager[] getTrustManagers(KeyStore trustStore, Collection<? extends CRL> crls) throws Exception {
+        TrustManager[] trustManagers = super.getTrustManagers(trustStore, crls);
+
+        if (trustManagers == null) {
+            return null;
+        }
+
+        for (int i = 0; i < trustManagers.length; ++i) {
+            if (X509TrustManager.class.isAssignableFrom(trustManagers[i].getClass())) {
+                X509TrustManagerWithLogs x509TrustManagerWithLogs = new X509TrustManagerWithLogs((X509TrustManager) trustManagers[i]);
+                trustManagers[i] = x509TrustManagerWithLogs;
+            }
+        }
+
+        return trustManagers;
+    }
+
+    public static class X509TrustManagerWithLogs implements X509TrustManager {
```

Comment:
X509TrustManagerWithLogs wrapps the X509TrustManager with the same methods only it print the certs on error

Reply:
Can we print it to sslcon.log?

Reply:
It does print it to sslconn.log. You'll notice the log is initiated like so:
    private static final Logger LOG = LoggerFactory.getLogger("SSLConnectionLog");